packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3294,6 +3295,17 @@ export interface VideoStreamRendererViewState {
 // @public
 export const VideoTile: (props: VideoTileProps) => JSX.Element;
 
+// @beta
+export interface VideoTileContextualMenuProps {
+    type: 'contextual';
```

Comment:
suggestion: `kind` instead of `type` - type is already a keyword in javascript so typically tend to avoid naming variables the same as keywords